boilerplate/scripts/build.js

```diff
@@ -1,8 +1,8 @@
 import builder from "statinamic/lib/builder"
 
-import * as layouts from "../web_modules/layouts"
-import metadata from "../web_modules/app/metadata"
-import routes from "../web_modules/app/routes"
+const layouts = require.resolve("../web_modules/layouts")
+const metadata = require.resolve("../web_modules/app/metadata")
+const routes = require.resolve("../web_modules/app/routes")
 import store from "../web_modules/app/store"
```

Comment:
The store only keep page data. Collection (during development) are in another memory cache.